settings/src/main/java/org/odk/collect/settings/ODKAppSettingsImporter.kt

```diff
@@ -0,0 +1,33 @@
+package org.odk.collect.settings
+
+import org.odk.collect.projects.Project
+import org.odk.collect.projects.ProjectsRepository
+import org.odk.collect.settings.importing.ProjectDetailsCreatorImpl
+import org.odk.collect.settings.importing.SettingsChangeHandler
+import org.odk.collect.settings.importing.SettingsImporter
+import org.odk.collect.settings.validation.JsonSchemaSettingsValidator
+
+class ODKAppSettingsImporter(
```

Comment:
Once I'd moved everything over, I realized that the constructor for `SettingsImporter` needed items that could now be internal to `settings`. Instead of spending time reworking that I've just wrapped `SettingsImporter` in a new, more convenient API. As noted in the PR description, there is more work we could do here.